test/e2e/simple_test.go

```diff
@@ -35,6 +35,8 @@ import (
 	"github.com/knative/build/pkg/apis/build/v1alpha1"
 )
 
+const defaultBuildTimeout = 120 * time.Second
```

Comment:
I think it might be better to keep this inlined in the test, unless you plan to update all the tests to specify this timeout value. Otherwise, "default" might be a bit confusing, since it's only actually used in one place.

Reply:
OK. Thanks! The timeout is now inlined.